package.json

```diff
@@ -6,6 +6,7 @@
   "dependencies": {
     "@mdx-js/mdx": "1.6.22",
     "@mdx-js/react": "1.6.22",
+    "@opentelemetry/instrumentation": "^0.30.0",
```

Comment:
Why have you added this to the docs pkg.json?

Reply:
I'm not sure what happened there, and was going to investigate. Anyway, I'll remove it.